datalad_osf/remote.py

```diff
@@ -23,29 +23,49 @@ class OSFRemote(SpecialRemote):
        Write doc section on how to do that, and what URL should be used to
        configure the special remote.
 
+
     Initialize the special remote::
 
-       git annex initremote osf type=external externaltype=osf \\
+       git annex initremote dolphinarchive type=external externaltype=osf \\
             encryption=none project=https://osf.io/<your-component-id>/
 
-    However, you may reuse an existing project without overwhelming it with
-    garbled filenames by setting a path where git-annex will store its data::
+    Authenticate (optionally):
 
-       git annex initremote osf type=external externaltype=osf \\
-            encryption=none project=https://osf.io/<your-component-id>/ \\
-            objpath=git-annex
+    The special remote is usable in a read-only form when configured like this,
+    which is perfectly good for most scientific applications.  But to upload a
+    dataset in the first place you need to supply a credential. Generate a login
+    token at https://osf.io/settings/tokens, giving it "osf.full_read" and
+    "osf.full_write" scopes.  Reconfigure the remote like so::
 
-    To upload files you need to supply credentials.
+       OSF_TOKEN='uo6g2MInjPRZ09P6Sl4qxX2TnwR3mFA1pMugZ9H1OMp6FtLefrbCxOTIvYn3gFmFOM2Fqx' \\
+            git annex enableremote dolphinarchive
 
-    .. todo::
+    You can also hand out read-only tokens to enable private read-only datasets. Create
+    tokens with only "osf.full_read" and give them to your collaborators.
```

Comment:
As far as I understood, OSF tokens are for users not projects. Giving out a read-only token, means giving read-only access to all of one's data on OSF. Am I misreading the situation?

Reply:
Good point. I would probably do it in a pinch to get a bunch of people up to speed but recommending it as general practice isn't a good idea.